reconcile/test/endpoints_discovery/test_endpoints_discovery_integration.py

```diff
@@ -68,14 +68,13 @@ def test_endpoints_discovery_integration_get_apps(
     intg: EndpointsDiscoveryIntegration,
 ) -> None:
     apps = intg.get_apps(query_func)
-    assert len(apps) == 6
+    assert len(apps) == 8
```

Comment:
is there a test to check 8 apps, but only N routes because of ignore?

Reply:
This includes the ignored ones, too. The `process test` tests the actual MR and route changes